src/Components/Users.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import UsersList from './Users_List';
+import styles from './css/style.css';
+
+const Users = () => {
+    console.log(process.env.REACT_APP_USERS_URL);
+    const URL = `${process.env.REACT_APP_USERS_URL}`;
+    //const URL = 'https://randomuser.me/api/?results=5';
+    
+    const[isLoading, setIsLoading] = useState(false);
+    const [users, setUsers] = useState([]);
+    const [error, setError] = useState(null);
+    
+    const fetchList = () => {
+        setIsLoading(true);
+        fetch(URL)
+            .then(res => {
+                if (res.ok) {
+                    setError(null);
+                    return res.json();
+                }
+                else
+                    throw Error(`Error fetching the API!`);
+            })
+            .then(apiData => {
+                const userDataList = [];
+                apiData.results.forEach(user => {
```

Comment:
When we want an array of values with the same number of items with an array we already have, the best function choice is the map function